EnglishFilter/sections/general_extensions.txt

```diff
@@ -232,6 +232,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75680
+imagetwist.com#%#//scriptlet("abort-current-inline-script", "document.attachEvent", "initBCPopunder")
```

Comment:
What kind of popups? Not reproduced on my end with US IP, but I can see it on the code. Can you provide an example via screenshot?

Reply:
@zloyden Check with AdGuard For Windows and disable AdGuard Browser Extension. Then click on the logo, you should see a redirect to `livejasmin.com` and the legitimate website in the next tab.

Reply:
Weird. let me double-check it.

Reply:
Ok, we can add it.

Reply:
Okay. @zloyden Is it possible to write HTML rule from here?

Reply:
The scriptlet is enough here.

Reply:
I am just asking. I won't add it.

Reply:
Yes, it's possible if you mean `$$` rule. It also works here, [but](https://github.com/AdguardTeam/AdguardFilters/pull/75743#discussion_r581914744).

Reply:
@zloyden How to count the characters for min length and max length?